recipes/extra-cmake-modules/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.0)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+
+find_package(ECM REQUIRED NO_MODULE)
+set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
```

```suggestion
list(APPEND CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
```

or should it be `list(INSERT CMAKE_MODULE_PATH 0 ${ECM_MODULE_PATH})` ?